lib/interface_quda.cpp

```diff
@@ -38,6 +38,12 @@
 #endif
 #include <gauge_update_quda.h>
 
+#if defined(NVSHMEM_COMMS)
```

Comment:
`shmem_helper.h`

Reply:
and following on from my comment below, this nvshmem include probably isn't even needed here since the nvshmem code should be moved to `targets/cuda/`

Reply:
Thoughts on whether to include that in `device.cpp` or a separate `comms.cpp` file?

Reply:
I think a separate file would be the way to go here.